recipes/cocoyaxi/all/conanfile.py

```diff
@@ -89,8 +89,8 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = ["co"]
-        self.cpp_info.names["cmake_find_package"] = "co"
-        self.cpp_info.names["cmake_find_package_multi"] = "co"
+        self.cpp_info.names["cmake_find_package"] = "cocoyaxi"
+        self.cpp_info.names["cmake_find_package_multi"] = "cocoyaxi"
```

Comment:
```suggestion
```

your library doesn't export CMake config files, so do not suggest in `package_info()` that it does. But I would recommend to add this feature in your CMakeLists, it's very useful for consumers.